### ISSUE TITLE
Remove 'oss' prefix from package naming

### ISSUE THREAD

**Issue by mnunberg** [RedisLabsModules/RLTest#4]

**Comment by mnunberg** [comment]
This is probably a helpful tool to use for the community as well, and calling something 'oss' makes it seem a bit less community-friendly.

**Comment by MeirShpilraien** [comment]
what should we change it to?

**Comment by mnunberg** [comment]
non-prefixed should mean vanilla OSS, and prefixed should be ‘enterprise’, or ‘E’, or ‘RL’, etc.— basically. And maybe put it in a separate directory as well.
